docs/source/subcommands.md

```diff
@@ -20,13 +20,9 @@ One of `r`, `conda`, `subprocess`, `empty`. If not provided, user will be prompt
 
 Generate a Dockerfile for the workflow instead of relying on [auto-generation.](basics/defining_environment.md#automatic-dockerfile-generation)
 
-#### `--cuda`
+#### `--base-image`, `-b`
 
-Make cuda drivers available to task code.
-
-#### `--opencl`
-
-Make opencl drivers available to task code.
+One of `opencl`, `cuda`, `docker`, `default`. If provided, the workflow will be built using with the specified software in the environment. These options are mutually exclusive. If not provided or `default`, will use a bare-minimum image.
```

Comment:
- `If provided` is the most redundant thing ever
- `specialized` is a weird value judgement
- use the explanation that I wrote in `docs/source/basics/defining_environment.md`, it's more descriptive than what's here
- ideally even delete it from that file to avoid redundancy and just leave a link to here
```suggestion
One of `opencl`, `cuda`, `docker`, or `default`. INSERT EXPLANATION FROM THE OTHER FILE
```